index.bs

```diff
@@ -1450,6 +1472,7 @@ typedef (WebGLRenderingContext or
          WebGL2RenderingContext) XRWebGLRenderingContext;
 
 dictionary XRWebGLLayerInit {
```

Comment:
Should we throw an exception if useDefaultFramebuffer is true and framebuffferScaleFactor isn't 1?

Reply:
If we do that we'd basically want to state that when this flag is present nothing else can be specified, because it should all defer to the GL context attributes. I'm hesitant to say that it should throw, though, because if we add new options in the future they would then start throwing on newer implementations and being ignored on older implementations. It seems more consistent to just say that everything else is ignored when this flag is present.

Reply:
You're totally right about framebufferScaleFactor.  We specified it to be ignorable by the UA which is why it's an attribute on the layer (so developers can know if it was actually applied).

Ok, but then what does that mean about the other flags that actually do defer to the actual GL context attributes?  Are they reported on the gl context itself?  If so, then yeah we can probably gracefully ignore the invalid data.  Probably worth explicitly stating that in the spec though so no one tries to implement a system that attempts to change those properties.

Reply:
They are reported on the GL context. Via `gl.getContextAttributes()` specifically. I'll add some text about it.

Reply:
Did this get added and I missed it?

Reply:
It did get addressed but I apparently didn't push it? Sorry! Change in question was https://github.com/immersive-web/webxr/pull/656/commits/f363b89c6ed504295667df8d3f1be6df86597749